README.md

```diff
@@ -1,5 +1,19 @@
 # ffd-info-exchange
 
+## About this project
+
+Welcome to the ffd-info-exchange repository! The Federal Front Door: Information Exchange project is a twelve-week discovery engagement meant to uncover possibilities for information sharing between federal agencies as a way of providing better public access to government services. 
```

Comment:
I like that you mention the duration. Perhaps add the end date ("a twelve-week discovery engagement ending ___" or otherwise)?